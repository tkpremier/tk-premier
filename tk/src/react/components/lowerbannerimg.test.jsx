import React from 'react';
import LowerBannerImg from './lowerbannerimg';
import renderer from 'react-test-renderer';

it('should render a lower banner image on hover', () => {
    const lowerbannerimg = renderer.create(
        <LowerBannerImg
          id="1"
          desktopImageUrl="https://8675fa3b12fbaae97df1-49af760ec3cce3b92480c70a3569b570.ssl.cf2.rackcdn.com/HomePage/lowerBanner-1-desktop.jpg"
          mobileImageUrl="https://8675fa3b12fbaae97df1-49af760ec3cce3b92480c70a3569b570.ssl.cf2.rackcdn.com/HomePage/lowerBanner-1-mobile.jpg"
          url="/auctions/auction/CH080117"
          type="lowerBanner"
        />
    );
    const tree = lowerbannerimg.toJSON();
    expect(tree).toMatchSnapshot();
});
