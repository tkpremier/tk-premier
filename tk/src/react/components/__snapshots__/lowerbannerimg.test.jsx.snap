// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a lower banner image on hover 1`] = `
<section
  id="lower-banner"
>
  <a
    href="/auctions/auction/CH080117"
    onClick={[Function]}
  >
    <picture>
      <source
        media="(min-width: 480px)"
        srcSet="https://8675fa3b12fbaae97df1-49af760ec3cce3b92480c70a3569b570.ssl.cf2.rackcdn.com/HomePage/lowerBanner-1-desktop.jpg"
      />
      <img
        alt="â€¦"
        srcSet="https://8675fa3b12fbaae97df1-49af760ec3cce3b92480c70a3569b570.ssl.cf2.rackcdn.com/HomePage/lowerBanner-1-mobile.jpg"
      />
    </picture>
  </a>
</section>
`;
